lib/Github/Api/PullRequest.php

```diff
@@ -90,9 +90,14 @@ public function commits($username, $repository, $id)
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/commits');
     }
 
-    public function files($username, $repository, $id)
+    public function files($username, $repository, $id, array $params = [])
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/files');
+        $parameters = array_merge([
+            'page' => 1,
```

Comment:
I don't think we should set the default values, let github decide on them. Just allow users to override these parameters

Reply:
@acrobat sounds fair, done